src/auxiliary/t8code.jl

```diff
@@ -254,20 +332,175 @@ function trixi_t8_fill_mesh_info(forest, elements, interfaces, mortars, boundari
 
                     # One-based indexing.
                     boundaries.name[boundary_id] = boundary_names[iface + 1, itree + 1]
+
+                else # Interfaces/mortars.
+                    neighbor_level = t8_element_level(neighbor_scheme, neighbor_leafs[1])
+
+                    # Local interfaces/mortars.
+                    if all(neighbor_ielements .< num_local_elements)
+                        # Conforming interface: The second condition ensures we only visit the interface once.
+                        if level == neighbor_level && current_index <= neighbor_ielements[1]
+                            local_num_conform += 1
+
+                            interfaces.neighbor_ids[1, local_num_conform] = current_index +
+                                                                            1
+                            interfaces.neighbor_ids[2, local_num_conform] = neighbor_ielements[1] +
+                                                                            1
```

Comment:
```suggestion
                            interfaces.neighbor_ids[1, local_num_conform] = current_index + 1
                            interfaces.neighbor_ids[2, local_num_conform] = neighbor_ielements[1] + 1
```
Not sure if this is valid with formatting though.